akka-persistence/src/test/scala/akka/persistence/SnapshotRecoveryLocalStoreSpec.scala

```diff
@@ -22,9 +22,10 @@ object SnapshotRecoveryLocalStoreSpec {
   }
 
   class LoadSnapshotTestPersistentActor(name: String, probe: ActorRef) extends NamedPersistentActor(name)
-    with TurnOffRecoverOnStart
     with ActorLogging {
 
+    override def recovery = Recovery(toSequenceNr = 0)
```

Comment:
ah, the test is storing a snapshot before any events have been persisted, i.e. seqNr = 0
ok, that would be weird usage in a real application, but I guess it's fine here.